pkg/metrics/collector.go

```diff
@@ -328,6 +349,12 @@ func buildMetricDescriptors(config trivyoperator.ConfigData) metricDescriptors {
 		configAuditLabels,
 		nil,
 	)
+	configAuditInfoDesc := prometheus.NewDesc(
+		prometheus.BuildFQName("trivy", "configaudits", "info"),
+		"Number of failing resource configuration auditing checks",
```

Comment:
it preferred to add the `info` at the end of description and fix the test , to distinguish between `info` and `summary`

Reply:
Alright Let me add me it now